18.md

```diff
@@ -19,6 +19,12 @@ to indicate where it can be fetched.
 The repost SHOULD include a `p` tag with the `pubkey` of the event being
 reposted.
 
+## Reposting Parameterized Replaceable Events
+
+An `a` tag with `<kind>:<pubkey>:<d-identifier>`
+MAY also be included in the repost event for client to retrieve the most recent
+version of the parameterized replaceable event(see [NIP-33](https://github.com/nostr-protocol/nips/blob/master/33.md)) being reposted.
```

Comment:
When reposting events of kind `30000 <= n < 40000`, a parameterised replaceable event as defined by [NIP-33](https://github.com/nostr-protocol/nips/blob/master/33.md), the repost SHOULD also include an `a` tag of the form `<kind>:<pubkey>:<d-identifier>`. This allows the most recent version of the parameterised event to be retrieved.